diff --git a/misc/quitcd/quitcd.bash_sh_zsh b/misc/quitcd/quitcd.bash_sh_zsh
index 79e7ea4c..5a160d9f 100644
--- a/misc/quitcd/quitcd.bash_sh_zsh
+++ b/misc/quitcd/quitcd.bash_sh_zsh
@@ -11,7 +11,7 @@ n ()
     # see. To cd on quit only on ^G, remove the "export" and make sure not to
     # use a custom path, i.e. set NNN_TMPFILE *exactly* as follows:
     #      NNN_TMPFILE="${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.lastd"
-    export NNN_TMPFILE="${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.lastd"
+    NNN_TMPFILE="${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.lastd"
 
     # Unmask ^Q (, ^V etc.) (if required, see `stty -a`) to Quit nnn
     # stty start undef
